templates/omit.d.ts

```diff
@@ -1,3 +1,8 @@
+import { Omit } from './$operation';
 import { List, Property } from './$types';
 
-export function $<T>(keys: List<Property>, object: T): Partial<T>;
+export function $string<T, K extends keyof T>(
```

Comment:
Not sure if we should add a `Record` version (i.e. `$record<T extends Record<K, any>, K extends string>`), if `K` is inferred as `string` then the result will be `never`, for example:

![image](https://user-images.githubusercontent.com/8341033/33157183-25e90ccc-d03b-11e7-82f8-4685362b8553.png)

Reply:
guess we can't use `keyof` in curried positions, though otherwise that might help a bit...

I fear type widening may yield `string` pretty easily, though somehow the other examples here seem to be working. weird.

anyway, `Pick<object, never>` evaluates to `{}`. I feel like we're gaining more info with the `Pick` over `Partial` than we're losing to that `{}`. sounds like a net gain to me, so let's try it?